src/streaming/text/TextTracks.js

```diff
@@ -657,24 +679,59 @@ function TextTracks(config) {
         return cue;
     }
 
+    function _getCueInformation(currentItem, timeOffset) {
+        if (currentItem.type === 'html') {
+            let cue = new Cue(currentItem.start + timeOffset, currentItem.end + timeOffset, '');
```

Comment:
One suggestion here: Can we move the logic in this function into two additional functions to not violate the single responsibility principle:

* `getCueInformationForHtml`
* `getCueInformationForNonHtml`: Probably not the best name as very similar to the name of the function above but ok for this case.